src/core/index.ts

```diff
@@ -57,13 +63,80 @@ export const journey = wrapFnWithLocation(
   }
 );
 
-export const step = wrapFnWithLocation(
+journey.skip = wrapFnWithLocation(
```

Comment:
Pretty much duplicated. Both of the methods .skip and .only can be invoked by having a common function that does the same thing. 
```ts
func addAnnotation(name: 'only' | 'skip' , args) { 
    const j = new Journey(...args);
    j[name]= true;
    runner.addJourney(j);
}

journey.only = addAnnotation('only')
journey.skip = addAnnotation('skip')

```

Reply:
done